code/modules/spells/spell_types/spacetime_distortion.dm

```diff
@@ -73,6 +76,7 @@
 /obj/effect/cross_action/spacetime_dist/New()
```

Comment:
Yes. Except instead of loc, the first argument to Initialize is mapload, true if it's being initialized on mapload